Calibration/HcalAlCaRecoProducers/plugins/AlCaLowPUHBHEMuonFilter.cc

```diff
@@ -179,7 +179,7 @@ bool AlCaLowPUHBHEMuonFilter::filter(edm::Event& iEvent, edm::EventSetup const&
               << RecMuon->innerTrack().isNonnull() << " outerTrack " << RecMuon->outerTrack().isNonnull()
               << " globalTrack " << RecMuon->globalTrack().isNonnull();
 #endif
-          if ((RecMuon->pt() < minimumMuonpT_) || fabs(RecMuon->eta() < minimumMuoneta_))
+          if ((RecMuon->pt() < minimumMuonpT_) || (RecMuon->eta() < minimumMuoneta_))
```

Comment:
```suggestion
          if ((RecMuon->pt() < minimumMuonpT_) || std::abs(RecMuon->eta()) < minimumMuoneta_)
```
this seems more correct in this context

Reply:
as the comment says :) (but I wasn't sure)